composer.json

```diff
@@ -30,17 +30,16 @@
     },
     "require": {
         "php": ">=5.5.0",
-        "symfony/symfony": "~2.2",
+        "symfony/symfony": "~2.7",
         "sensio/framework-extra-bundle": "^3.0"
     },
     "require-dev": {
-        "behat/behat": ">=2.2.2",
+        "behat/behat": "~2.5",
         "kriswallsmith/buzz": ">=0.5",
         "phpunit/phpunit": "4.8.*",
         "behat/symfony2-extension": "*@stable",
         "behat/mink-extension": "*",
         "behat/mink-browserkit-driver": "*",
-        "behat/sahi-client": "v1.0.1",
```

Comment:
Installed as a dependency of the next line, removing the explicit (and utterly precise) version fixes a conflict.